contributing.md

```diff
@@ -19,6 +19,7 @@ When in doubt on how to organize your example, start by copying the `examples/te
 1. Clone the project down to your computer.
 2. From the project folder, run `npm install` in your terminal.
 3. Start a local static server with `npm start` in your terminal and go to `localhost:8080` in your browser to see the site.
```

Comment:
I know you didn't write this, but let's just remove the `and go to `localhost:8080` in your browser to see the site` since the port might be different.